### PR TITLE
Create README

### PR DESCRIPTION
**Pull request by taymoork2**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+
+## Install
+
+```bash
+npm install --save @webex/component-adapters
+```
+
+## Usage
+
+_Happy Coding!_
```
Comment (adamweeks) [inline README.md]: Webex Component Adapter Interfaces are utilized when implementing a (link needed) Webex Component Adapter. 

An example of extending the RoomsAdapter for your datasource:
 
```suggestion
import {RoomsAdapter} from '@webex/components-adapter-interfaces';

export default class MyRoomsAdapter extends RoomsAdapter {
  getRoom(ID) {
   // ... my implemenation
  }
}
```

File: README.md [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+  - [Release Process](#release-process)
+    - [Commit linter](#commit-linter)
+- [License](#license)
+- [Support](#support)
+- [Team](#team)
+
+## Install
+
+```bash
+npm install --save @webex/component-adapters
```
Comment (adamweeks) [inline README.md]:
```suggestion
npm install --save @webex/component-adapters-interfaces
```

File: README.md [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+  </a>
+  <a href='#badge'>
+    <img alt='semantic-release' src='https://img.shields.io/badge/%20%20%F0%9F%93%A6%F0%9F%9A%80-semantic--release-e10079.svg'>
+  </a>
+  <a href='https://github.com/webex/component-adapter-interfaces/blob/master/package.json#L28'>
+    <img src='https://img.shields.io/npm/l/webex.svg' alt='license'>
+  </a>
+</p>
+
+**Webex Component Adapters** is a set of adapters...
```
Comment (adamweeks) [inline README.md]:
```suggestion
**Webex Component Adapter Interfaces** are a set of adapter interfaces which are utilized when implementing a <link needed> Webex Component Adapter.
```

File: README.md [hunk 4]
```diff
@@ -0,0 +1,104 @@
...
+  - [Release Process](#release-process)
+    - [Commit linter](#commit-linter)
+- [License](#license)
+- [Support](#support)
+- [Team](#team)
+
+## Install
+
+```bash
+npm install --save @webex/component-adapters-interfaces
```
Comment (adamweeks) [inline README.md]:
```suggestion
npm install --save @webex/component-adapter-interfaces
```

File: README.md [hunk 5]
```diff
@@ -0,0 +1,104 @@
...
+   // ... my implemenation
+  }
+}
+```
+
+_Happy Coding!_
+
+## Contributing
+
+We'd love for you to contribute to our source code and to make **Webex Component Adapters** even better than it is today! Here are some [guidelines](https://github.com/webex/components/blob/master/CONTRIBUTING.md) that we'd like you to follow.
```
Comment (adamweeks) [inline README.md]: `Webex Component Adapter Interfaces`

File: README.md [hunk 6]
```diff
@@ -0,0 +1,104 @@
+<h1 align='center' style='border-bottom: none;'>Webex Component Adapters</h1>
```
Comment (adamweeks) [inline README.md]:
```suggestion
<h1 align='center' style='border-bottom: none;'>Webex Component Adapter Interfaces</h1>
```

File: README.md [hunk 7]
```diff
@@ -0,0 +1,104 @@
...
+<h3 align='center'>Library of Adapters for Webex components</h3>
```
Comment (adamweeks) [inline README.md]:
```suggestion
<h3 align='center'>Library of Adapter Interfaces for Webex components</h3>
```

File: README.md [hunk 8]
```diff
@@ -0,0 +1,104 @@
...
+  </a>
+  <a href='#badge'>
+    <img alt='semantic-release' src='https://img.shields.io/badge/%20%20%F0%9F%93%A6%F0%9F%9A%80-semantic--release-e10079.svg'>
+  </a>
+  <a href='https://github.com/webex/component-adapter-interfaces/blob/master/package.json#L28'>
+    <img src='https://img.shields.io/npm/l/webex.svg' alt='license'>
+  </a>
+</p>
+
+**Webex Component Adapter Interfaces** is a set of interfaces...
```
Comment (lalli-flores) [inline README.md]:
```suggestion
**Webex Component Adapter Interfaces** is a set of interfaces that Webex Components (<= make this a link to the repo) use as a contract on how to obtain data.
```

File: README.md [hunk 9]
```diff
@@ -0,0 +1,104 @@
...
+
+[MIT License](https://opensource.org/licenses/MIT)
+
+## Support
+
+For more developer resources, tutorials and support, visit the Webex developer portal, https://developer.webex.com.
+
+## Team
+
+| [![Adam Weeks](https://github.com/adamweeks.png?size=100)](https://github.com/adamweeks) | [![Arash Koushkebaghi](https://github.com/akoushke.png?size=100)](https://github.com/akoushke) | [![Lalli Flores](https://github.com/lalli-flores.png?size=100)](https://github.com/lalli-flores) | [![Timothy Scheuering](https://github.com/InteractiveTimmy.png?size=100)](https://github.com/InteractiveTimmy) | [![David Hoff](https://github.com/harborhoffer.png?size=100)](https://github.com/harborhoffer) |
```
Comment (lalli-flores) [inline README.md]: uhm, do you wanna add yourself to the team list? 😅

Comment (akoushke) [reply README.md]: Copy paste this:

Comment (akoushke) [reply README.md]: ## Team

| [Adam Weeks](https://github.com/adamweeks) | [Arash Koushkebaghi](https://github.com/akoushke) | [Lalli Flores](https://github.com/lalli-flores) | [Timothy Scheuering](https://github.com/InteractiveTimmy) | [David Hoff](https://github.com/harborhoffer) | [Taymoor Khan](https://github.com/taymoork2) |
| ---------------------------------------------------------------------------------------- | ---------------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------ | -------------------------------------------------------------------------------------------------------------- | ---------------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------ |
| [Adam Weeks](https://github.com/adamweeks)                                               | [Arash Koushkebaghi](https://github.com/akoushke)                                              | [Lalli Flores](https://github.com/lalli-flores)                                                  | [Timothy Scheuering](https://github.com/InteractiveTimmy)                                                      | [David Hoff](https://github.com/harborhoffer)                                                  | [Taymoor Khan](https://github.com/taymoork2)                                               |

Comment (akoushke) [reply README.md]: make sure it looks good though. Preview it.

File: README.md [hunk 10]
```diff
@@ -0,0 +1,104 @@
...
+  </a>
+  <a href='#badge'>
+    <img alt='semantic-release' src='https://img.shields.io/badge/%20%20%F0%9F%93%A6%F0%9F%9A%80-semantic--release-e10079.svg'>
+  </a>
+  <a href='https://github.com/webex/component-adapter-interfaces/blob/master/package.json#L28'>
+    <img src='https://img.shields.io/npm/l/webex.svg' alt='license'>
+  </a>
+</p>
+
+**Webex Component Adapter Interfaces** is a set of interfaces that Webex Components (<= make this a link to the repo) use as a contract on how to obtain data.
```
Comment (adamweeks) [inline README.md]:
```suggestion
**Webex Component Adapter Interfaces** is a set of interfaces that [Webex Components](https://github.com/webex/components#adapters) use as a contract on how to obtain data.
```

**Pull request merged**